src/components/Dashboard.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import  '@testing-library/jest-dom'
+
+import Dashboard from './Dashboard';
+
+
+
+afterEach(cleanup);
+
+describe('Dashboard.js', () => {
+    
+    it('displays counts in each function', () =>{
+        const { getByTestId } = render(<Dashboard />)
+        expect(getByTestId('count')).toHaveTextContent('0')
```

Comment:
I believe this is failing because the jest-dom stuff is installed incorrectly. Doing the same test without the jest-dom selectors passes:         
```suggestion
        expect(queryByTestId('count').textContent).toBe('0')
```